apps/src/templates/currentUserRedux.js

```diff
@@ -185,6 +186,13 @@ export default function currentUser(state = initialState, action) {
       user_type,
       experiments.getEnabledExperiments()
     );
+    // Calling Statsig separately to emphasize different user integrations
+    // and because dual reporting is aspirationally temporary (March 2024)
+    statsigReporter.setUserProperties(
```

Comment:
This function is async- will this have any impact on currentUser redux running effectively?

Reply:
This is starting to push my JS knowledge a bit but it looks like async functions implicitly [return a Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function). So, I _think_ that this is fine for this redux store.

Reply:
I was able to move this through to fix the console errors and didn't notice any lag in loadtime. Marking resolved